DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -567,34 +625,33 @@ void TrackAnalyzer::bookHistosForHitProperties(DQMStore::IBooker & ibooker) {
         oriAlgo->setBinLabel(ibin+1,reco::TrackBase::algoNames[ibin]);
       }
 
-      // DataFormats/TrackReco/interface/TrajectoryStopReasons.h
-      std::vector<std::string> StopReasonName = { "UNINITIALIZED", "MAX_HITS", "MAX_LOST_HITS", "MAX_CONSECUTIVE_LOST_HITS", "LOST_HIT_FRACTION", "MIN_PT", "CHARGE_SIGNIFICANCE", "LOOPER", "MAX_CCC_LOST_HITS", "NO_SEGMENTS_FOR_VALID_LAYERS", "SEED_EXTENSION", "NOT_STOPPED" };
-      if(StopReasonName.size() != static_cast<unsigned int>(StopReason::SIZE)) {
+      size_t StopReasonNameSize = sizeof(StopReasonName::StopReasonName)/sizeof(std::string);
+      if(StopReasonNameSize != static_cast<unsigned int>(StopReason::SIZE)) {
         throw cms::Exception("Assert") << "StopReason::SIZE is " << static_cast<unsigned int>(StopReason::SIZE)
-                                       << " but TrackAnalyzer has StopReasonName's only for "
-                                       << StopReasonName.size()
-                                       << ". Please update also TrackAnalyzer.cc near line " << __LINE__ << ".";
+                                       << " but StopReasonName's only for "
+                                       << StopReasonNameSize
+                                       << ". Please update DataFormats/TrackReco/interface/TrajectoryStopReasons.h.";
```

Comment:
This check would be better implemented as a `static_assert` in DataFormats/TrackReco/interface/TrajectoryStopReasons.h